### ISSUE TITLE
Bridge.Browser checks not working

### ISSUE THREAD

**Issue by Danielku15** [bridgedotnet/Bridge#765]
It seems the browser detection mechansim of Bridge.Browser is quite broken. I did some basic tests and all fail: 

## Chrome 47.0.2526 Windows 10 - 64 bit
UserAgent: Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.106 Safari/537.36
Expected: isChrome=true, isDesktop=true, isPhone=false, isWindows=true, 
Actual: isChrome=false, isPhone=true , isDesktop=false, isWindows=false, 

## IE 11 Windows 10 - 64 bit
UserAgent: Mozilla/5.0 (Windows NT 10.0; WOW64; Trident/7.0; .NET4.0C; .NET4.0E; .NET CLR 2.0.50727; .NET CLR 3.0.30729; .NET CLR 3.5.30729; rv:11.0) like Gecko
Expected: isIE=true, isIE11=true, isDesktop=true, isPhone=false, isWindows=true, 
Actual: isIE=false, isIE11=false, isDesktop=false, isPhone=true, isWindows=false, 

## Firefox 43.0.1 Windows 10 - 64 bit
UserAgent: Mozilla/5.0 (Windows NT 10.0; WOW64; rv:43.0) Gecko/20100101 Firefox/43.0
Expected: isGecko=true, isDesktop=true, isPhone=false, isWindows=true, 
Actual: isGecko=false, isDesktop=false, isPhone=true, isWindows=false, 

It seems the major issue is the case sensitive check of the user agent which breaks most of the checks. .

**Comment by geoffreymcgill** [comment]
Hi @Danielku15, 

Thanks for contributing the fix.

Your patch is simple, although before accepting the pull-request we require your acceptance of the Bridge CLA. You can submit the CLA form at the following location:

http://goo.gl/forms/CvDFLA3EPO

More information on contributing to Bridge can be found at the following location:

https://github.com/bridgedotnet/Bridge/blob/master/CONTRIBUTING.md

**Comment by Danielku15** [comment]
Wow, that was a quick response. Of course I am willing to agree your CLA but I am a bit concerned about the "Mailing Address" field or your acceptance form. I am not willing to share my real name *and* real address as part of an open source project contribution, but I've submitted the form without this information. 

Regarding this issue: I prepared a pull request (#766) regarding the fix. I'm not quite sure about your test infrastructure for such cross browser tests. 

According to the source itself it would anyhow require some more changes to test the behavior based on a given input value and not fixed to `navigator.userAgent`
